src/DynamicData.Tests/API/ApiApprovalBase.cs

```diff
@@ -15,29 +21,125 @@ namespace DynamicData.APITests
     [UsesVerify]
     public abstract class ApiApprovalBase
     {
+        private static readonly Regex _removeCoverletSectionRegex = new(@"^namespace Coverlet\.Core\.Instrumentation\.Tracker.*?^}", RegexOptions.Singleline | RegexOptions.Multiline | RegexOptions.Compiled);
+
         /// <summary>
         /// Check Approval.
         /// </summary>
         /// <param name="assembly">The assembly.</param>
+        /// <param name="memberName">Name of the member.</param>
         /// <param name="filePath">The file path.</param>
-        /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
-        protected static Task CheckApproval(Assembly assembly, [CallerFilePath] string? filePath = null)
+        /// <returns>
+        /// A <see cref="Task" /> representing the result of the asynchronous operation.
```

Comment:
This doc is no longer valid.

Reply:
I have found the issue with the original code and updated, I was missing a Task and an await, many apologies.